libs/langchain-community/src/chat_models/bedrock/web.ts

```diff
@@ -680,31 +746,77 @@ export class BedrockChat extends BaseChatModel implements BaseBedrockInput {
   }
 
   override bindTools(
-    tools: (StructuredToolInterface | Record<string, unknown>)[],
-    _kwargs?: Partial<BaseChatModelCallOptions>
+    tools: this["ParsedCallOptions"]["tools"],
+    _kwargs?: Partial<this["ParsedCallOptions"]>
   ): Runnable<
     BaseLanguageModelInput,
     BaseMessageChunk,
-    BaseChatModelCallOptions
+    this["ParsedCallOptions"]
   > {
     const provider = this.model.split(".")[0];
     if (provider !== "anthropic") {
       throw new Error(
         "Currently, tool calling through Bedrock is only supported for Anthropic models."
       );
     }
-    this._anthropicTools = tools.map((tool) => {
-      if (isStructuredTool(tool)) {
-        return {
-          name: tool.name,
-          description: tool.description,
-          input_schema: zodToJsonSchema(tool.schema),
-        };
-      }
-      return tool;
-    });
+    this._anthropicTools = formatTools(tools);
     return this;
   }
+
+  withStructuredOutput<
```

Comment:
cc @jacoblee93 we still need to add this even though there is a base implementation since it's not a required method. If we wanted to remove it, we'd need to make it a required method, and add some sort of `throw new Error("Not implemented")` in every chat model class (which I'm against)